sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery/api/Azure.ResourceManager.SpringAppDiscovery.netstandard2.0.cs

```diff
@@ -0,0 +1,414 @@
+        public string AppType { get { throw null; } set { } }
+        public string ArtifactName { get { throw null; } set { } }
+        public System.Collections.Generic.IList<int> BindingPorts { get { throw null; } }
+        public string BuildJdkVersion { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> Certificates { get { throw null; } }
+        public string Checksum { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> ConnectionStrings { get { throw null; } }
+        public System.Collections.Generic.IList<string> Dependencies { get { throw null; } }
+        public System.Collections.Generic.IList<string> Environments { get { throw null; } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.SpringAppDiscovery.Models.Error> Errors { get { throw null; } }
+        public int? InstanceCount { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.SpringAppDiscovery.Models.SpringbootappsPropertiesInstancesItem> Instances { get { throw null; } }
+        public string JarFileLocation { get { throw null; } set { } }
+        public int? JvmMemoryInMB { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> JvmOptions { get { throw null; } }
+        public System.DateTimeOffset? LastModifiedOn { get { throw null; } set { } }
+        public System.DateTimeOffset? LastUpdatedOn { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> MachineArmIds { get { throw null; } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.SpringAppDiscovery.Models.SpringbootappsPropertiesMiscsItem> Miscs { get { throw null; } }
+        public Azure.ResourceManager.SpringAppDiscovery.Models.ProvisioningState? ProvisioningState { get { throw null; } set { } }
+        public string RuntimeJdkVersion { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> Servers { get { throw null; } }
+        public string SiteName { get { throw null; } set { } }
+        public string SpringBootVersion { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> StaticContentLocations { get { throw null; } }
+    }
+    public partial class SpringbootappsPropertiesApplicationConfigurationsItem
+    {
+        public SpringbootappsPropertiesApplicationConfigurationsItem(string key) { }
+        public string Key { get { throw null; } set { } }
+        public string Value { get { throw null; } set { } }
+    }
+    public partial class SpringbootappsPropertiesInstancesItem
+    {
+        public SpringbootappsPropertiesInstancesItem(string machineArmId) { }
+        public int? InstanceCount { get { throw null; } set { } }
+        public int? JvmMemoryInMB { get { throw null; } set { } }
+        public string MachineArmId { get { throw null; } set { } }
```

Comment:
the name suggests this property is an arm resource id, please confirm. If so, please change the format of this property to `arm-id`

Reply:
It is just a string to identify a marchine arm id(not including a arm resource format), so will keep here